library/hawkular_alerts_group_trigger.py

```diff
@@ -260,6 +326,7 @@ def main():
     enabled    = module.params['enabled']
     verify_ssl = module.params['verify_ssl']
     ca_file    = module.params['ca_file_path']
+    conditions = module.params['group_conditions']
```

Comment:
Aha, `group_conditions` includes `trigger_mode` and `conditions` list.
Though I don't see an actual reason why they _must_ be different names.

Do you really need the nested structure?
If you have just one `trigger_mode`, it _sounds_ like it belongs directly on the trigger, and then you could simply have a `conditions` list at lop level?  What am I missing?